test/integration/https.spec.ts

```diff
@@ -419,4 +419,73 @@ describe("When configured for HTTPS", () => {
             });
         });
     });
-});
\ No newline at end of file
+
+    describe.only("with TLS version restrictions", () => {
```

Comment:
This shouldn't be `.only`, or it'll disable all the other tests!

Reply:
nice catch! artefact from my feedback loop of implementing this